### PR TITLE
Desktop App uninstall

### PR DESCRIPTION
**Pull request by cwarnermm**
Documentation for: https://mattermost.atlassian.net/browse/MM-48999

### CODE REVIEW

**Review by oh6hay — APPROVED** [review]

File: source/install/desktop-app-install.rst [hunk 1]
```diff
@@ -210,6 +210,10 @@ The location of the Mattermost desktop app configuration file depends on the pla
 - macOS App Store: ``/Users/<username>/Library/Containers/Mattermost/Data/Library/Application Support/Mattermost``
 - Linux: ``~/.config/Mattermost``
 
+.. note::
+
+  If you choose to uninstall the desktop app, you must manually remove local configuration data from the applicable location noted above.
```
Comment (jupenur) [inline source/install/desktop-app-install.rst:215]:
```suggestion
  Local configuration data is not automatically removed when uninstalling the desktop app. If you wish to remove all data, you must manually remove the files from the applicable location noted above.
```

**Review by jupenur — COMMENTED** [review]
Added a wording suggestion: deleting local data is not a "must" in every scenario; it only is if you want to make sure the app is completely gone and don't wish a fresh install to use any of your old settings.

I'm also wondering if we should add another note along these lines:

> Prior to uninstalling, users may also choose to log out of any active sessions. Active sessions may also be terminated from another Mattermost session by going to **Profile** > **Security** > **View and Logout of Active Sessions** and selecting **Log Out**. Desktop app sessions are labeled as **Native Desktop App**.

**Comment by cwarnermm** [comment]
Thanks, @jupenur! I've incorporated your feedback in another commit.

**Review by jupenur — APPROVED** [review]

File: source/install/desktop-app-install.rst [hunk 2]
```diff
@@ -210,6 +210,11 @@ The location of the Mattermost desktop app configuration file depends on the pla
 - macOS App Store: ``/Users/<username>/Library/Containers/Mattermost/Data/Library/Application Support/Mattermost``
 - Linux: ``~/.config/Mattermost``
 
+.. note::
+
+  - Local configuration data is not automatically removed when uninstalling the desktop app. If you wish to remove all data, you must manually remove the files from the applicable location noted above.
+  - Prior to uninstalling, you can choose to log out of any active sessions. Active sessions may also be terminated from another Mattermost session by going to **Profile > Security > View and Logout of Active Sessions**, and selecting **Log Out**. Desktop app sessions are labeled as **Native Desktop App**.
```
Comment (justinegeffen) [inline source/install/desktop-app-install.rst:216]:
```suggestion
  - Prior to uninstalling, you can choose to log out of any active sessions. You can terminate active sessions from another Mattermost session in **Profile > Security > View and Logout of Active Sessions**, then select **Log Out**. Desktop app sessions are labeled as **Native Desktop App**.
```

**Review by justinegeffen — APPROVED** [review]

**Pull request merged**